console/src/components/Services/Events/CronTriggers/PendingEvents/PendingEvents.tsx

```diff
@@ -19,6 +25,34 @@ const PendingEvents: React.FC<Props> = props => {
   const { dispatch, currentTrigger } = props;
   const triggerName = currentTrigger ? currentTrigger.name : '';
 
+  const onCancelCronTrigger = (id: string, onSuccess: () => void) => {
```

Comment:
Ditto, this logic is duplicated. We could have a thunk in `serverIO` that takes `table name` as an argument.